js/addToTask.js

```diff
@@ -90,20 +90,23 @@ TRASH_INPUT.addEventListener(CLICK_EVENT, function (event) {
   INPUT.value = null;
 });
 
-let removeListener = false;
-
 function onActionTodo(event) {
   const element = event.target;
   const elementJob = element.getAttribute("data-job");
   if (elementJob === DELETE_TODO) {
     removeTodo(element.parentNode, taskList, Number(element.id));
   } else if (elementJob === COMPLETE) {
     markTodoAsCompleted(element.parentNode, taskList, Number(element.id));
+  } else if (elementJob === EDIT) {
+    editTask(element.parentNode, taskList, Number(element.id));
   }
-  removeListener = true;
-}
 
-UL_LIST.addEventListener(CLICK_EVENT, onActionTodo);
-if (removeListener) {
   UL_LIST.removeEventListener(CLICK_EVENT, onActionTodo);
+  addTodoClickListener();
```

Comment:
event listener should not be removed right after it is registered. consider removing when the associated DOM node `UL_LIST` is removed.